crates/derive/src/stages/batch_queue.rs

```diff
@@ -476,6 +662,7 @@ mod tests {
         let mut bq = BatchQueue::new(cfg, mock, fetcher);
         let parent = L2BlockInfo::default();
         let result = bq.next_batch(parent).await;
-        assert!(result.is_err());
+        assert!(result.is_ok());
+        // assert_eq!(result, Err(StageError::NotEnoughData));
```

Comment:
What's going on here?

Reply:
Oops, there were a few comments from a bad merge I believe. I've removed them and cleaned this up.